package.json

```diff
@@ -22,5 +22,9 @@
     "@mermaid-js/mermaid-cli": "^11.4.2",
     "@types/bun": "latest",
     "typescript": "^5"
+  },
+  "dependencies": {
+    "@raydium-io/raydium-sdk-v2": "^0.1.127-alpha",
+    "@solana/spl-token": "^0.4.13"
```

Comment:
Should be in the backend/package.json file, not here.
There must be a way to specify the project to use when doing `bun i` but I haven't checked too much. For now I just go to the specific project folder, so in this case, backend, and run the `bun I @solana/spl-token` for example.